### ISSUE TITLE
More implementations

### ISSUE THREAD

**Issue by mimoo** [mimoo/disco#4]
To become useful, Disco needs to be widely supported. This means a libdisco needs to exist in:

* C for embedded devices. Possibly using [tweetFIPS202](https://github.com/jedisct1/TweetFIPS202) and other small implementations.
* Rust
* Erlang
* Java
* etc.

Furthermore, it needs to be integrated in load balancers and proxies:

* HAProxy
* Apache
* Nginx

**Comment by mimoo** [comment]
**How to implement Disco?**

1. Implement or find an implementation of [Strobe](https://strobe.sourceforge.io/). (The reference code is in C and python.)
1. Read the "How to Read This Document and Implement Disco" section of the [Disco spec](http://discocrypto.com/disco.html). What it tells you is to implement [Noise](http://noiseprotocol.org/noise.html) but to ignore its SymmetricState and CipherState sections.

>

**Comment by Fasjeit** [comment]
I've just started working on [python](https://github.com/Fasjeit/PyDisco) version.

 ... better late than never :)

Actually noting (except a couple of symmetric methods) but there will be more in the nearest future.

**Comment by mimoo** [comment]
yeah! That's awesome :) I'll take a look at it when I get some time.

**Comment by mimoo** [comment]
Looks like there is [a swift version](https://github.com/nixberg/GDisco)

**Comment by mimoo** [comment]
there's Xisco which doesn't use Strobe but xoodyak! https://github.com/nixberg/Xisco

**Comment by mimoo** [comment]
And a rust implementation: https://github.com/rust-p2p/disco
